models/article-models.js

```diff
@@ -27,7 +27,13 @@ exports.selectArticles = (query) => {
 };
 
 exports.selectArticle = (article_id) => {
-    return db.query(format(`SELECT * FROM articles WHERE article_id = %L;`, [article_id]))
+    return db.query(format(`
+        SELECT articles.*, COUNT(comments.comment_id) AS comment_count 
+        FROM articles 
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        WHERE articles.article_id = %L

```

Comment:
%L may not be the most appropriate way of inserting the article id here.

Reply:
Thanks, I have now updated endpoints & submitted a new commit